Assets/Plugins/Colyseus/Connection.cs

```diff
@@ -25,21 +27,30 @@ private void Initialize()
 			OnClose += _OnClose;
 		}
 
-		public async Task Send(object[] data)
+		public async Task Send(byte[] data)
 		{
-			var serializationOutput = new MemoryStream();
-			MsgPack.Serialize(data, serializationOutput, SerializationOptions.SuppressTypeInformation);
+			if (!IsOpen) {
+				_enqueuedCalls.Enqueue(data);
```

Comment:
can't this initial queue be removed if the messages before the _OnOpen call happed also land in the base's message queue? _OnOpen starts the ProcessMessageQueue Task that basically does exactly the same as the loop at L62.

if the async task is not desiredfor webgl, _OnOpen can call DispatchMessageQueue to resend the queued messages ( basically what happens @L62) ?